pkg/collector/corechecks/containerimage/check.go

```diff
@@ -149,7 +149,11 @@ func (c *Check) Run() error {
 
```

Comment:
can we do `defer c.processor.stop()` instead of calling the function in the `for {}` loop when the `case` exit for the loop?